tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/AWS/LambdaOneParam.cs

```diff
@@ -0,0 +1,53 @@
+// <copyright file="LambdaOneParam.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.ComponentModel;
+
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.ServerlessInstrumentation.AWS
+{
+    /// <summary>
+    /// Lambda customer handler calltarget instrumentation
+    /// </summary>
+    [Browsable(false)]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class LambdaOneParam
+    {
+        private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor(typeof(LambdaOneParam));
+
+        /// <summary>
+        /// OnMethodBegin callback
+        /// </summary>
+        /// <typeparam name="TTarget">Type of the target</typeparam>
+        /// <param name="instance">Instance value, aka `this` of the instrumented method.</param>
+        /// <param name="incomingEventOrContext">IncomingEventOrContext value</param>
+        /// <returns>Calltarget state value</returns>
+        internal static CallTargetState OnMethodBegin<TTarget>(TTarget instance, object incomingEventOrContext)
```

Comment:
is `incomingEventOrContext` really of type `object`? If not, I recommend to add another Generic parameter (eg. `TArg1`)